completions/java

```diff
@@ -103,15 +96,15 @@ _comp_cmd_java__classes()
 # available packages completion
 _comp_cmd_java__packages()
 {
-    local sourcepath i
+    local ret i
 
     # find which sourcepath to use
-    _java_find_sourcepath
+    _comp_cmd_java__find_sourcepath
 
     # convert package syntax to path syntax
     cur=${cur//.//}
     # parse each sourcepath element for packages
-    for i in ${sourcepath//:/ }; do
+    for i in ${ret//:/ }; do
```

Comment:
These changes overlap with #888. I'll later check both changes.

Reply:
Yeah, that one's marked as draft so I wasn't quite sure of its status. But we can drop the java changes from here if you like, and address them later/elsewhere.

Reply:
Removed java from here, added pkgutil, rpm, and ssh instead.